src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -1060,7 +1060,8 @@ async function processDataType(
 function prepareNumber(a, dataType) {
   return {
     size: a.$.size,
-    is_signed: a.$.name.endsWith('u') || !a.$.name.includes('int') ? 0 : 1,
+    is_signed:
+      a.$.name.endsWith('u') || !/int[0-9]{1,2}/g.test(a.$.name) ? 0 : 1,
```

Comment:
How about:
```suggestion
      /^int[0-9]{1,2}$/.test(a.$.name) ? 1 : 0,
```

Reply:
> How about:

I have seen xml go with either `int8s` or `int8 for signed types

Reply:
In that case `/^int[0-9]{1,2}s?$/.test(a.$.name) ? 1 : 0`.

Reply:
Hopefully that does not break something in the Zigbee pro stack. I can try it.

Reply:
Added more logic to handle any future exceptions that can arise from types such as temperature, etc.
Just mentioned signed="true" in the xml and that will load them correctly.